src/app/pages/rpc-stx-call-contract/rpc-stx-call-contract-container.tsx

```diff
@@ -36,7 +36,7 @@ export function RpcStxCallContractContainer() {
                 nonce={nonce}
                 onGoBack={() => navigate(RouteUrls.RpcStxCallContract)}
               >
-                <RpcTransactionRequestProvider<RpcCallContractRequestContext>
+                <RpcTransactionRequestProvider<RpcStxCallContractRequestContext>
```

Comment:
Given that we don't utilise a return type value for this component, I wonder if using a generic is the right pattern to use here.

Alternatively, couldn't we make a higher order component, or new providers for each flow, with the common behaviours like all the state in a generic hook?

Reply:
I do think my initial setup sharing the provider ended up being a bit overkill, so I'll simplify it. 👍